addons/zeus/CfgEventHandlers.hpp

```diff
@@ -4,3 +4,11 @@ class Extended_PreInit_EventHandlers {
         init = QUOTE(call COMPILE_FILE(XEH_preInit));
     };
 };
+
+class Extended_InitPost_EventHandlers {
+    class AllVehicles {
+        class ADDON {
+            init = QUOTE(call FUNC(addObjectToCurator));
```

Comment:
Also, we can actually just make this serverInit since it only need to run server side.